docs/overview.md

```diff
@@ -0,0 +1,100 @@
+# Cumulus overview
```

Comment:
```suggestion
# Cumulus Overview
```

I will be nitpicky/opinionated with my review, feel free to take what you like.
 
Sections/Title headers usually follow https://titlecase.com/, which means most letters are capitalized.